app/services/problem_check/email_polling_errored_recently.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ProblemCheck::EmailPollingErroredRecently < ProblemCheck
+  self.priority = "low"
+
+  def call
+    return no_problem if !(polling_error_count > 0)
```

Comment:
Would this be easier to read? 🤔 

```suggestion
    return no_problem if polling_error_count <= 0
```

Reply:
I think easier to read, harder to understand. 😅

Reply:
I'm gonna take the plunge and change it to `== 0`.